src/main/java/org/isf/patient/gui/PatientGuiConst.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2021 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isf.patient.gui;
+
+public interface PatientGuiConst {
+
+	public static final Integer IMAGE_THUMBNAIL_MAX_WIDTH = 140;
```

Comment:
In addition, there is a compilation error in one of the checks (Java CI with Maven / build (push) Failing after 1m )

Reply:
That is because this PR is paired with https://github.com/informatici/openhospital-core/pull/721, that has already been merged, so `pull_request` check is ok, `push` check cannot.